README.md

```diff
@@ -289,6 +289,58 @@ export GITHUB_TOKEN=$(gh auth token)
 binst check
 ```
 
+## 📋 Schema Command
+
+The `binst schema` command displays the binstaller configuration schema directly from the CLI in various formats. This is useful for understanding the configuration structure, IDE integration, and automated tooling.
+
+### Basic Usage
+
+```bash
+# Display schema in YAML format (default)
+binst schema
+
+# Display schema in JSON format  
+binst schema --format json
+
+# Display original TypeSpec source
+binst schema --format typespec
+```
+
+### Filtering with External Tools
+
+The schema command is designed to work seamlessly with external tools like `yq` and `jq` for filtering and processing:
+
+```bash
+# List all available schema types
+binst schema | yq '."$defs" | keys'
+
+# Filter specific type definitions
+binst schema | yq '."$defs".AssetConfig'
+binst schema | yq '."$defs".Platform'
+
+# Get only the root schema (without type definitions)
+binst schema | yq 'del(."$defs")'
+
+# Use jq for JSON processing
+binst schema --format json | jq '."$defs".Binary'
+binst schema --format json | jq '.properties.supported_platforms'
```

Comment:
this command just shows reference and it's not useful.
Let's remove it.

Instead, it's actually helpful to tell users how to get list of supported platform os/arch. Please provide both jq/yq command example.